server/routes/state.ts

```diff
@@ -38,3 +38,30 @@ export const ensureImprisonmentStatusPresentMiddleware =
   (req: Request, res: Response, next: NextFunction) => {
     return isStatePresent(COOKIE_NAME)(req) ? next() : res.redirect(redirectUrl)
   }
+
+export const SexCodec: Codec<string> = {
+  write: (value: string): Record<string, string> => {
+    return {
+      data: value,
+    }
+  },
+
+  read(record: Record<string, unknown>): string {
+    assertHasStringValues(record, ['data'])
+    return record.data
+  },
+}
+
+const SEX_COOKIE_NAME = 'sex'
+
+export const clearSex = clearState('sex')
```

Comment:
Can't see where this is used? 

Need to do where we do `clearImprisonmentStatus` in both `confirmAddedToRollController` and `confirmArrivalController`.

Otherwise a user answering that once for one arrival will use that answer for all future arrivals